src/Makefile.in

```diff
@@ -80,7 +80,10 @@ FAST_TIME_PRELOAD=@FAST_TIME_PRELOAD@
 
 TARGETS=$(LIBMTEV) $(LIBMTEV_MAJOR) $(FAST_TIME_PRELOAD) @LUA_LUAMTEV@ @MDB_MODS@
 
-all:	reversion $(TARGETS) make-man build-modules
+all:	reversion $(TARGETS) make-man build-modules ex
+
+ex:
```

Comment:
spell out `examples` here?

Reply:
`examples` is the name of the directory which throws make under the bus.